src/SWP/FixturesBundle/Resources/themes/theme_1/index.html.twig

```diff
@@ -7,4 +7,11 @@
     <a href="{{ path('swp_page_about_us') }}">About Us</a>
     <a href="{{ path('swp_page_features') }}">Features</a>
     <a href="{{ path('swp_page_get_involved') }}">Get Involved</a>
+
+    <h2>News</h2>
+    <ul>
+    {% gimmelist article from articles with {'pageName': 'News'} %}
+        <li><a href="{{ generateUrlForArticle(article) }}">{{ article.title }} </a></li>
```

Comment:
this function should be shorter imho, like `urlFor($article)`, `gimmeUrl($article)` etc., you already check the instance of passed param in that Extension's function. You pass instance of Article as a param, so you already know what you want to get, typing `..Article(article)` makes no sense to me.